backend/main/chapters/c06_lists.py

```diff
@@ -93,9 +93,9 @@ def solution(self, words: List[str]):
             (['The', 'quick', 'brown', 'fox', 'jumps'], 'Thequickbrownfoxjumps'),
         ]
 
-    class double_numbers(ExerciseStep):
+    class strings_sum_bonus(ExerciseStep):
         """
-Optional bonus challenge: extend the program to insert a separator string *between* each word.
+Optional Bonus challenge: extend the program to insert a separator string *between* each word.
```

Comment:
To make an exercise optional, users need to be able to skip past it easily. Don't make them open the table of contents, provide a link to the next page. See the existing bonus challenge.

Reply:
OK, added links to both.